app/src/main/java/org/mozilla/fenix/settings/ToolbarSettingsFragment.kt

```diff
@@ -5,49 +5,46 @@
 package org.mozilla.fenix.settings
 
 import android.os.Bundle
-import androidx.appcompat.app.AppCompatActivity
 import androidx.preference.PreferenceFragmentCompat
 import org.mozilla.fenix.R
 import org.mozilla.fenix.components.metrics.Event
 import org.mozilla.fenix.ext.components
 import org.mozilla.fenix.ext.getPreferenceKey
+import org.mozilla.fenix.ext.showToolbar
 
+/**
+ * Settings to adjust the position of the browser toolbar.
+ */
 class ToolbarSettingsFragment : PreferenceFragmentCompat() {
-    private lateinit var topPreference: RadioButtonPreference
-    private lateinit var bottomPreference: RadioButtonPreference
 
     override fun onCreatePreferences(savedInstanceState: Bundle?, rootKey: String?) {
         setPreferencesFromResource(R.xml.toolbar_preferences, rootKey)
     }
 
     override fun onResume() {
         super.onResume()
-        (activity as AppCompatActivity).title = getString(R.string.preferences_toolbar)
-        (activity as AppCompatActivity).supportActionBar?.show()
+        showToolbar(getString(R.string.preferences_toolbar))
 
         setupClickListeners()
-        setupRadioGroups()
     }
 
     private fun setupClickListeners() {
```

Comment:
nit: consider renaming this to `setupPreferences` or `setupClickListenersAndRadioGroups` since it now does both?